libs/lib-client/header/src/lib/header.component.ts

```diff
@@ -15,22 +22,13 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
  *            right side block
  *
  *  - bottom block (selector: `header-role="bottom-block"`): This will allow users add any additional items to the bottom of
- *            the header. For example, in flows, the flow and error tabs are added to the bottom of the header
+ *            the header. It is mandatory to add #bottomBlock reference to the bottom component area.
```

Comment:
what happened with the directive approach? that would eliminate the need for another config for the user